docs/operations/asset-repository.md

```diff
@@ -0,0 +1,60 @@
+# Using local asset repositories
+
+You can configure kOps to provision a cluster to download assets (images and files) from local repositories.
+This is useful when downloading assets from the internet is undersirable, for example:
+
+* To deploy where the network is offline or internet-restricted.
+* To avoid rate limits or network transfer costs.
+* To limit exposure to watering-hole attacks.
+* To comply with other security requirements, such as the need to scan for vulnerabilities.
+
+There can be one repository for images and another for files.
+
+## Configuring
+
+### Configuring a local image repository
+
+To configure a local image repository, set either `assets.containerRegistry` or `assets.containerProxy` in the cluster spec.
+They both do essentially the same thing, but `containerRegistry` avoids using `/` characters in the local image names.
+
+```yaml
+spec:
+  assets:
+    containerRegistry: example.com/registry
+```
+
+or
+
+```yaml
+spec:
+  assets:
+    containerProxy: example.com/proxy
+```
+
+### Configuring a local file repository
+
+To configure a local file repository, set `assets.fileRepository` in the cluster spec.
+
+```yaml
+spec:
+  assets:
+    fileRepository: https://example.com/files
+```
+
+## Copying assets into repositories
+
+You can copy assets into their repositories either by running `kops update cluster --phase assets --create-kube-config=false --yes` or through an external process.
```

Comment:
Any idea when this was added? It is not what I would expect.
This is more of a `toolbox` thing than `update cluster` thing.